tests/components/scene/fog.test.js

```diff
@@ -2,16 +2,20 @@
 var entityFactory = require('../../helpers').entityFactory;
 
 suite('fog', function () {
-  'use strict';
-
-  setup(function () {
+  setup(function (done) {
     this.entityEl = entityFactory();
     var el = this.el = this.entityEl.parentNode;
-    this.updateMaterialsSpy = this.sinon.spy(el.systems.material, 'updateMaterials');
+    var self = this;
+
+    el.addEventListener('loaded', function () {
```

Comment:
this is in a test, there are over 700 other instances in the tests of unnamed functions that just wait for loaded.